### ISSUE TITLE
sourcetrail fails to update

### ISSUE THREAD

**Issue by aminya** [digitalcoyote/chocolatey-packages#7]
Source trail fails to update with this error
![image](https://user-images.githubusercontent.com/16418197/86325096-20d66000-bc05-11ea-9f52-f9833a6d4219.png)

**Comment by digitalcoyote** [comment]
It will probably be tomorrow night before I can look at this. I assume this was installed/updated with the Chocolatey GUI?

**Comment by aminya** [comment]
Yes! Let me know if you need more info, but I think this is reproducible.

**Comment by digitalcoyote** [comment]
I wasn't able to reproduce it on my windows VM. Any chance you can send me the log for that (be sure to double check it for sensitive data).

**Comment by aminya** [comment]
No worries. Here is the log:
[logs.zip](https://github.com/digitalcoyote/chocolatey-packages/files/4867214/logs.zip)


```ps1
❯ choco upgrade sourcetrail -y -v
```
output:
```ps1
Chocolatey v0.10.15
Upgrading the following packages:
sourcetrail
By upgrading you accept licenses for the packages.

You have sourcetrail v2020.2.37 installed. Version 2020.2.43 is available based on your source(s).
Progress: Downloading sourcetrail 2020.2.43... 100%
[NuGet] Uninstalling 'sourcetrail 2020.2.37'.
[NuGet] Successfully uninstalled 'sourcetrail 2020.2.37'.
[NuGet] Installing 'sourcetrail 2020.2.43'.
[NuGet] Successfully installed 'sourcetrail 2020.2.43'.

sourcetrail v2020.2.43 [Approved]
sourcetrail package files upgrade completed. Performing other installation steps.
VERBOSE: Exporting function 'Format-FileSize'.
VERBOSE: Exporting function 'Get-ChecksumValid'.
VERBOSE: Exporting function 'Get-ChocolateyUnzip'.
VERBOSE: Exporting function 'Get-ChocolateyWebFile'.
VERBOSE: Exporting function 'Get-EnvironmentVariable'.
VERBOSE: Exporting function 'Get-EnvironmentVariableNames'.
VERBOSE: Exporting function 'Get-FtpFile'.
VERBOSE: Exporting function 'Get-OSArchitectureWidth'.
VERBOSE: Exporting function 'Get-PackageParameters'.
VERBOSE: Exporting function 'Get-PackageParametersBuiltIn'.
VERBOSE: Exporting function 'Get-ToolsLocation'.
VERBOSE: Exporting function 'Get-UACEnabled'.
VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
VERBOSE: Exporting function 'Get-VirusCheckValid'.
VERBOSE: Exporting function 'Get-WebFile'.
VERBOSE: Exporting function 'Get-WebFileName'.
VERBOSE: Exporting function 'Get-WebHeaders'.
VERBOSE: Exporting function 'Install-BinFile'.
VERBOSE: Exporting function 'Install-ChocolateyDesktopLink'.
VERBOSE: Exporting function 'Install-ChocolateyEnvironmentVariable'.
VERBOSE: Exporting function 'Install-ChocolateyExplorerMenuItem'.
VERBOSE: Exporting function 'Install-ChocolateyFileAssociation'.
VERBOSE: Exporting function 'Install-ChocolateyInstallPackage'.
VERBOSE: Exporting function 'Install-ChocolateyPackage'.
VERBOSE: Exporting function 'Install-ChocolateyPath'.
VERBOSE: Exporting function 'Install-ChocolateyPinnedTaskBarItem'.
VERBOSE: Exporting function 'Install-ChocolateyPowershellCommand'.
VERBOSE: Exporting function 'Install-ChocolateyShortcut'.
VERBOSE: Exporting function 'Install-ChocolateyVsixPackage'.
VERBOSE: Exporting function 'Install-ChocolateyZipPackage'.
VERBOSE: Exporting function 'Install-Vsix'.
VERBOSE: Exporting function 'Set-EnvironmentVariable'.
VERBOSE: Exporting function 'Set-PowerShellExitCode'.
VERBOSE: Exporting function 'Start-ChocolateyProcessAsAdmin'.
VERBOSE: Exporting function 'Test-ProcessAdminRights'.
VERBOSE: Exporting function 'Uninstall-BinFile'.
VERBOSE: Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.
VERBOSE: Exporting function 'Uninstall-ChocolateyPackage'.
VERBOSE: Exporting function 'Uninstall-ChocolateyZipPackage'.
VERBOSE: Exporting function 'Update-SessionEnvironment'.
VERBOSE: Exporting function 'Write-ChocolateyFailure'.
VERBOSE: Exporting function 'Write-ChocolateySuccess'.
VERBOSE: Exporting function 'Write-FileUpdateLog'.
VERBOSE: Exporting function 'Write-FunctionCallLogMessage'.
VERBOSE: Exporting alias 'Get-ProcessorBits'.
VERBOSE: Exporting alias 'Get-OSBitness'.
VERBOSE: Exporting alias 'Get-InstallRegistryKey'.
VERBOSE: Exporting alias 'Generate-BinFile'.
VERBOSE: Exporting alias 'Add-BinFile'.
VERBOSE: Exporting alias 'Start-ChocolateyProcess'.
VERBOSE: Exporting alias 'Invoke-ChocolateyProcess'.
VERBOSE: Exporting alias 'Remove-BinFile'.
VERBOSE: Exporting alias 'refreshenv'.
VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-core\chocolatey-core.psm1'.
VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
VERBOSE: Exporting function 'Get-AppInstallLocation'.
VERBOSE: Exporting function 'Get-AvailableDriveLetter'.
VERBOSE: Exporting function 'Get-EffectiveProxy'.
VERBOSE: Exporting function 'Get-PackageCacheLocation'.
VERBOSE: Exporting function 'Get-PackageParameters'.
VERBOSE: Exporting function 'Get-WebContent'.
VERBOSE: Exporting function 'Register-Application'.
VERBOSE: Exporting function 'Remove-Process'.
VERBOSE: Importing function 'Get-AppInstallLocation'.
VERBOSE: Importing function 'Get-AvailableDriveLetter'.
VERBOSE: Importing function 'Get-EffectiveProxy'.
VERBOSE: Importing function 'Get-PackageCacheLocation'.
VERBOSE: Importing function 'Get-PackageParameters'.
VERBOSE: Importing function 'Get-UninstallRegistryKey'.
VERBOSE: Importing function 'Get-WebContent'.
VERBOSE: Importing function 'Register-Application'.
VERBOSE: Importing function 'Remove-Process'.
VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-dotnetfx\chocolatey-dotnetfx.psm1'.
VERBOSE: Exporting function 'Install-DotNetFramework'.
VERBOSE: Exporting function 'Install-DotNetDevPack'.
VERBOSE: Importing function 'Install-DotNetDevPack'.
VERBOSE: Importing function 'Install-DotNetFramework'.
VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-font-helpers\FontHelp.psm1'.
VERBOSE: Exporting function 'Install-ChocolateyFont'.
VERBOSE: Exporting function 'Uninstall-ChocolateyFont'.
VERBOSE: Exporting alias 'Add-Font'.
VERBOSE: Exporting alias 'Remove-Font'.
VERBOSE: Importing function 'Install-ChocolateyFont'.
VERBOSE: Importing function 'Uninstall-ChocolateyFont'.
VERBOSE: Importing alias 'Add-Font'.
VERBOSE: Importing alias 'Remove-Font'.
VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-visualstudio\chocolatey-visualstudio.extension.psm1'.
VERBOSE: Exporting function 'Add-VisualStudioComponent'.
VERBOSE: Exporting function 'Add-VisualStudioWorkload'.
VERBOSE: Exporting function 'Get-VisualStudioInstaller'.
VERBOSE: Exporting function 'Get-VisualStudioInstallerHealth'.
VERBOSE: Exporting function 'Get-VisualStudioInstance'.
VERBOSE: Exporting function 'Get-VisualStudioVsixInstaller'.
VERBOSE: Exporting function 'Install-VisualStudio'.
VERBOSE: Exporting function 'Install-VisualStudioInstaller'.
VERBOSE: Exporting function 'Install-VisualStudioVsixExtension'.
VERBOSE: Exporting function 'Remove-VisualStudioComponent'.
VERBOSE: Exporting function 'Remove-VisualStudioProduct'.
VERBOSE: Exporting function 'Remove-VisualStudioWorkload'.
VERBOSE: Exporting function 'Uninstall-VisualStudio'.
VERBOSE: Exporting function 'Uninstall-VisualStudioVsixExtension'.
VERBOSE: Importing function 'Add-VisualStudioComponent'.
VERBOSE: Importing function 'Add-VisualStudioWorkload'.
VERBOSE: Importing function 'Get-VisualStudioInstaller'.
VERBOSE: Importing function 'Get-VisualStudioInstallerHealth'.
VERBOSE: Importing function 'Get-VisualStudioInstance'.
VERBOSE: Importing function 'Get-VisualStudioVsixInstaller'.
VERBOSE: Importing function 'Install-VisualStudio'.
VERBOSE: Importing function 'Install-VisualStudioInstaller'.
VERBOSE: Importing function 'Install-VisualStudioVsixExtension'.
VERBOSE: Importing function 'Remove-VisualStudioComponent'.
VERBOSE: Importing function 'Remove-VisualStudioProduct'.
VERBOSE: Importing function 'Remove-VisualStudioWorkload'.
VERBOSE: Importing function 'Uninstall-VisualStudio'.
VERBOSE: Importing function 'Uninstall-VisualStudioVsixExtension'.
VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-windowsupdate\chocolatey-windowsupdate.psm1'.
VERBOSE: Exporting function 'Install-WindowsUpdate'.
VERBOSE: Exporting function 'Test-WindowsUpdate'.
VERBOSE: Importing function 'Install-WindowsUpdate'.
VERBOSE: Importing function 'Test-WindowsUpdate'.
VERBOSE: Exporting function 'Format-FileSize'.
VERBOSE: Exporting function 'Get-ChecksumValid'.
VERBOSE: Exporting function 'Get-ChocolateyUnzip'.
VERBOSE: Exporting function 'Get-ChocolateyWebFile'.
VERBOSE: Exporting function 'Get-EnvironmentVariable'.
VERBOSE: Exporting function 'Get-EnvironmentVariableNames'.
VERBOSE: Exporting function 'Get-FtpFile'.
VERBOSE: Exporting function 'Get-OSArchitectureWidth'.
VERBOSE: Exporting function 'Get-PackageParameters'.
VERBOSE: Exporting function 'Get-PackageParametersBuiltIn'.
VERBOSE: Exporting function 'Get-ToolsLocation'.
VERBOSE: Exporting function 'Get-UACEnabled'.
VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
VERBOSE: Exporting function 'Get-VirusCheckValid'.
VERBOSE: Exporting function 'Get-WebFile'.
VERBOSE: Exporting function 'Get-WebFileName'.
VERBOSE: Exporting function 'Get-WebHeaders'.
VERBOSE: Exporting function 'Install-BinFile'.
VERBOSE: Exporting function 'Install-ChocolateyDesktopLink'.
VERBOSE: Exporting function 'Install-ChocolateyEnvironmentVariable'.
VERBOSE: Exporting function 'Install-ChocolateyExplorerMenuItem'.
VERBOSE: Exporting function 'Install-ChocolateyFileAssociation'.
VERBOSE: Exporting function 'Install-ChocolateyInstallPackage'.
VERBOSE: Exporting function 'Install-ChocolateyPackage'.
VERBOSE: Exporting function 'Install-ChocolateyPath'.
VERBOSE: Exporting function 'Install-ChocolateyPinnedTaskBarItem'.
VERBOSE: Exporting function 'Install-ChocolateyPowershellCommand'.
VERBOSE: Exporting function 'Install-ChocolateyShortcut'.
VERBOSE: Exporting function 'Install-ChocolateyVsixPackage'.
VERBOSE: Exporting function 'Install-ChocolateyZipPackage'.
VERBOSE: Exporting function 'Install-Vsix'.
VERBOSE: Exporting function 'Set-EnvironmentVariable'.
VERBOSE: Exporting function 'Set-PowerShellExitCode'.
VERBOSE: Exporting function 'Start-ChocolateyProcessAsAdmin'.
VERBOSE: Exporting function 'Test-ProcessAdminRights'.
VERBOSE: Exporting function 'Uninstall-BinFile'.
VERBOSE: Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.
VERBOSE: Exporting function 'Uninstall-ChocolateyPackage'.
VERBOSE: Exporting function 'Uninstall-ChocolateyZipPackage'.
VERBOSE: Exporting function 'Update-SessionEnvironment'.
VERBOSE: Exporting function 'Write-ChocolateyFailure'.
VERBOSE: Exporting function 'Write-ChocolateySuccess'.
VERBOSE: Exporting function 'Write-FileUpdateLog'.
VERBOSE: Exporting function 'Write-FunctionCallLogMessage'.
VERBOSE: Exporting function 'Get-AppInstallLocation'.
VERBOSE: Exporting function 'Get-AvailableDriveLetter'.
VERBOSE: Exporting function 'Get-EffectiveProxy'.
VERBOSE: Exporting function 'Get-PackageCacheLocation'.
VERBOSE: Exporting function 'Get-WebContent'.
VERBOSE: Exporting function 'Register-Application'.
VERBOSE: Exporting function 'Remove-Process'.
VERBOSE: Exporting function 'Install-DotNetDevPack'.
VERBOSE: Exporting function 'Install-DotNetFramework'.
VERBOSE: Exporting function 'Install-ChocolateyFont'.
VERBOSE: Exporting function 'Uninstall-ChocolateyFont'.
VERBOSE: Exporting function 'Add-VisualStudioComponent'.
VERBOSE: Exporting function 'Add-VisualStudioWorkload'.
VERBOSE: Exporting function 'Get-VisualStudioInstaller'.
VERBOSE: Exporting function 'Get-VisualStudioInstallerHealth'.
VERBOSE: Exporting function 'Get-VisualStudioInstance'.
VERBOSE: Exporting function 'Get-VisualStudioVsixInstaller'.
VERBOSE: Exporting function 'Install-VisualStudio'.
VERBOSE: Exporting function 'Install-VisualStudioInstaller'.
VERBOSE: Exporting function 'Install-VisualStudioVsixExtension'.
VERBOSE: Exporting function 'Remove-VisualStudioComponent'.
VERBOSE: Exporting function 'Remove-VisualStudioProduct'.
VERBOSE: Exporting function 'Remove-VisualStudioWorkload'.
VERBOSE: Exporting function 'Uninstall-VisualStudio'.
VERBOSE: Exporting function 'Uninstall-VisualStudioVsixExtension'.
VERBOSE: Exporting function 'Install-WindowsUpdate'.
VERBOSE: Exporting function 'Test-WindowsUpdate'.
VERBOSE: Exporting alias 'Get-ProcessorBits'.
VERBOSE: Exporting alias 'Get-OSBitness'.
VERBOSE: Exporting alias 'Get-InstallRegistryKey'.
VERBOSE: Exporting alias 'Generate-BinFile'.
VERBOSE: Exporting alias 'Add-BinFile'.
VERBOSE: Exporting alias 'Start-ChocolateyProcess'.
VERBOSE: Exporting alias 'Invoke-ChocolateyProcess'.
VERBOSE: Exporting alias 'Remove-BinFile'.
VERBOSE: Exporting alias 'refreshenv'.
VERBOSE: Exporting alias 'Add-Font'.
VERBOSE: Exporting alias 'Remove-Font'.
VERBOSE: Importing function 'Add-VisualStudioComponent'.
VERBOSE: Importing function 'Add-VisualStudioWorkload'.
VERBOSE: Importing function 'Format-FileSize'.
VERBOSE: Importing function 'Get-AppInstallLocation'.
VERBOSE: Importing function 'Get-AvailableDriveLetter'.
VERBOSE: Importing function 'Get-ChecksumValid'.
VERBOSE: Importing function 'Get-ChocolateyUnzip'.
VERBOSE: Importing function 'Get-ChocolateyWebFile'.
VERBOSE: Importing function 'Get-EffectiveProxy'.
VERBOSE: Importing function 'Get-EnvironmentVariable'.
VERBOSE: Importing function 'Get-EnvironmentVariableNames'.
VERBOSE: Importing function 'Get-FtpFile'.
VERBOSE: Importing function 'Get-OSArchitectureWidth'.
VERBOSE: Importing function 'Get-PackageCacheLocation'.
VERBOSE: Importing function 'Get-PackageParameters'.
VERBOSE: Importing function 'Get-PackageParametersBuiltIn'.
VERBOSE: Importing function 'Get-ToolsLocation'.
VERBOSE: Importing function 'Get-UACEnabled'.
VERBOSE: Importing function 'Get-UninstallRegistryKey'.
VERBOSE: Importing function 'Get-VirusCheckValid'.
VERBOSE: Importing function 'Get-VisualStudioInstaller'.
VERBOSE: Importing function 'Get-VisualStudioInstallerHealth'.
VERBOSE: Importing function 'Get-VisualStudioInstance'.
VERBOSE: Importing function 'Get-VisualStudioVsixInstaller'.
VERBOSE: Importing function 'Get-WebContent'.
VERBOSE: Importing function 'Get-WebFile'.
VERBOSE: Importing function 'Get-WebFileName'.
VERBOSE: Importing function 'Get-WebHeaders'.
VERBOSE: Importing function 'Install-BinFile'.
VERBOSE: Importing function 'Install-ChocolateyDesktopLink'.
VERBOSE: Importing function 'Install-ChocolateyEnvironmentVariable'.
VERBOSE: Importing function 'Install-ChocolateyExplorerMenuItem'.
VERBOSE: Importing function 'Install-ChocolateyFileAssociation'.
VERBOSE: Importing function 'Install-ChocolateyFont'.
VERBOSE: Importing function 'Install-ChocolateyInstallPackage'.
VERBOSE: Importing function 'Install-ChocolateyPackage'.
VERBOSE: Importing function 'Install-ChocolateyPath'.
VERBOSE: Importing function 'Install-ChocolateyPinnedTaskBarItem'.
VERBOSE: Importing function 'Install-ChocolateyPowershellCommand'.
VERBOSE: Importing function 'Install-ChocolateyShortcut'.
VERBOSE: Importing function 'Install-ChocolateyVsixPackage'.
VERBOSE: Importing function 'Install-ChocolateyZipPackage'.
VERBOSE: Importing function 'Install-DotNetDevPack'.
VERBOSE: Importing function 'Install-DotNetFramework'.
VERBOSE: Importing function 'Install-VisualStudio'.
VERBOSE: Importing function 'Install-VisualStudioInstaller'.
VERBOSE: Importing function 'Install-VisualStudioVsixExtension'.
VERBOSE: Importing function 'Install-Vsix'.
VERBOSE: Importing function 'Install-WindowsUpdate'.
VERBOSE: Importing function 'Register-Application'.
VERBOSE: Importing function 'Remove-Process'.
VERBOSE: Importing function 'Remove-VisualStudioComponent'.
VERBOSE: Importing function 'Remove-VisualStudioProduct'.
VERBOSE: Importing function 'Remove-VisualStudioWorkload'.
VERBOSE: Importing function 'Set-EnvironmentVariable'.
VERBOSE: Importing function 'Set-PowerShellExitCode'.
VERBOSE: Importing function 'Start-ChocolateyProcessAsAdmin'.
VERBOSE: Importing function 'Test-ProcessAdminRights'.
VERBOSE: Importing function 'Test-WindowsUpdate'.
VERBOSE: Importing function 'Uninstall-BinFile'.
VERBOSE: Importing function 'Uninstall-ChocolateyEnvironmentVariable'.
VERBOSE: Importing function 'Uninstall-ChocolateyFont'.
VERBOSE: Importing function 'Uninstall-ChocolateyPackage'.
VERBOSE: Importing function 'Uninstall-ChocolateyZipPackage'.
VERBOSE: Importing function 'Uninstall-VisualStudio'.
VERBOSE: Importing function 'Uninstall-VisualStudioVsixExtension'.
VERBOSE: Importing function 'Update-SessionEnvironment'.
VERBOSE: Importing function 'Write-ChocolateyFailure'.
VERBOSE: Importing function 'Write-ChocolateySuccess'.
VERBOSE: Importing function 'Write-FileUpdateLog'.
VERBOSE: Importing function 'Write-FunctionCallLogMessage'.
VERBOSE: Importing alias 'Add-BinFile'.
VERBOSE: Importing alias 'Add-Font'.
VERBOSE: Importing alias 'Generate-BinFile'.
VERBOSE: Importing alias 'Get-InstallRegistryKey'.
VERBOSE: Importing alias 'Get-OSBitness'.
VERBOSE: Importing alias 'Get-ProcessorBits'.
VERBOSE: Importing alias 'Invoke-ChocolateyProcess'.
VERBOSE: Importing alias 'refreshenv'.
VERBOSE: Importing alias 'Remove-BinFile'.
VERBOSE: Importing alias 'Remove-Font'.
VERBOSE: Importing alias 'Start-ChocolateyProcess'.
File appears to be downloaded already. Verifying with package checksum to determine if it needs to be redownloaded.
Hashes match.
Hashes match.
Extracting C:\Temp\chocolatey\sourcetrail\2020.2.43\Sourcetrail_2020_2_43_Windows_64bit_Installer.zip to C:\ProgramData\chocolatey\lib\sourcetrail\Sourcetrail.zip...
VERBOSE:
VERBOSE:
VERBOSE: Extracting archive: C:\Temp\chocolatey\sourcetrail\2020.2.43\Sourcetrail_2020_2_43_Windows_64bit_Installer.zip
VERBOSE: 1 file, 95443227 bytes (92 MiB)
VERBOSE:
VERBOSE: Scanning the drive for archives:
VERBOSE: 7-Zip 18.06 (x86) : Copyright (c) 1999-2018 Igor Pavlov : 2018-12-30
VERBOSE: --
VERBOSE: Path = C:\Temp\chocolatey\sourcetrail\2020.2.43\Sourcetrail_2020_2_43_Windows_64bit_Installer.zip
VERBOSE: Type = zip
VERBOSE: Physical Size = 95443227
VERBOSE:
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\atom
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\atom\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\eclipse
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\eclipse\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\emacs
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\emacs\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\idea\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\qt_creator\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\sublime_text
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\idea
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\qt_creator
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\sublime_text\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\vim
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\visual_studio
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\visual_studio\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\vim\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\vscode
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\plugins\vscode\README.md
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\readme.txt
VERBOSE: - Sourcetrail_2020_2_43_64bit_Installer\sourcetrail.msi
VERBOSE: Everything is Ok
VERBOSE:
VERBOSE: Folders: 11
VERBOSE: Files: 11
VERBOSE: Size:       97351886
VERBOSE: Compressed: 95443227
ERROR: Cannot process argument transformation on parameter 'file'. Cannot convert value to type System.String.
Environment Vars (like PATH) have changed. Close/reopen your shell to
 see the changes (or in powershell/cmd.exe just type `refreshenv`).
The upgrade of sourcetrail was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\sourcetrail\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - sourcetrail (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\sourcetrail\tools\chocolateyinstall.ps1'.
 See log for details.

```

**Comment by digitalcoyote** [comment]
Not sure why I left this without a response, but I believe this was fixed in 890ec633080678d4cb0b58074b420212938ce7ba

**Comment by aminya** [comment]
Thanks!
